StreamChatUITestsAppUITests/Utilities/LaunchArgument.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct MockServerConfiguration {
+    static var port = Int.random(in: 61000..<62000)
+    static var websocketHost = "ws://localhost"
+    static var httpHost = "http://localhost"
+}
+
+enum EnvironmentVariable: String {
+
+    /// This changes the base url to localhost & assigns port when our LLC is built in Debug build configuration && `USE_MOCK_SERVER` launch argument is set.
```

Comment:
Would be nice to break down the comment in few lines, it's hardly readable like this.

Reply:
Thanks for the comment!. Will do:)

Reply:
Done ;)